src/stylesheets/layouts/_question.scss

```diff
@@ -0,0 +1,80 @@
+.question {
+  background-color: $color-mint;
+  display: flex;
+  justify-content: center;
+  margin-top: 1.5rem;
+
+  &__wrapper {
+    position: relative;
+    @include flex;
+    width: 100%;
+    height: 100%;
+    max-width: 77rem;
+    margin: 0rem;
+    padding: 2.5rem 1.5rem;
+
+    @media (min-width: 550px) {
+      margin: 2rem;
+      padding: 4rem 6.5rem;
+    }
+
+    @media (min-width: 800px) {
+      margin: 3rem;
+      padding: 5rem 8.5rem;
+    }
+  }
+
+
+  &__hdr {
+    margin-bottom: 1rem;
+  }
+
+  &__intro{
+    margin-bottom: $content-margin-btm;
+  }
+
+  &__context {
+    margin-bottom: $content-margin-btm;
+
+    @media (min-width: 550px) {
+      margin-right: 20rem;
+    }
+
+    @media (min-width: 550px) {
+      margin-right: 10rem;
+    }
```

Comment:
you have two `min-width: 550px` breakpoints here

Reply:
I'm curious how this `margin-right` comes into play

Reply:
is the margin leaving space for the phone?

Reply:
- deleted second margin. typo!!! resolved in pr#4
- yes, the space is for the phone. i use the margins and i have noticed that you use padding. is there a reason you use padding?